javascript/javascript2/week1/spiritAnim.js

```diff
@@ -0,0 +1,28 @@
+const spiritAnimalName = [
+  "The Tiger",
+  "The Hawk",
+  "Rattle Snake",
+  "Cobra",
+  "The Eagle",
+  "Black Lion",
+  "Flying Mantis",
+  "Crouching Tiger",
+  "Black Scorpion",
+  "Quick Tortoise"
+];
+const buttonTag = document.getElementById("clickonMe");
+buttonTag.addEventListener("click", spiritAnimal);
+
+function spiritAnimal() {
+  const inputName = document.getElementById("name"); //u can also put .value here
+  console.log("clicked");
+  if (!inputName.value) {
+    alert("give a valid name");
+  } else {
+    const randomName = Math.floor(Math.random() * spiritAnimalName.length);
+    const randomSpiritAnimal = spiritAnimalName[randomName];
+    document.getElementById(
+      "spiritAnim"
+    ).textContent = `${inputName.value} - ${randomSpiritAnimal}`;
+  }
+}
```

Comment:
So your code should be structured roughly like this. 
1. Get information from the DOM
2. Call a function that computes the spiritAnimal
3. Display that spirit animal by modifying the DOM